src/fomo.jl

```diff
@@ -1,11 +1,121 @@
-export fomo, FomoSolver, FoSolver, fo, R2, TR, tr_step, r2_step
+export fomo, FomoSolver, FOMOParameterSet, FoSolver, fo, R2, TR
+export tr_step, r2_step
 
 abstract type AbstractFirstOrderSolver <: AbstractOptimizationSolver end
 
 abstract type AbstractFOMethod end
 struct tr_step <: AbstractFOMethod end
 struct r2_step <: AbstractFOMethod end
 
+# Default algorithm parameter values
+const FOMO_η1 = "eps(T)^(1 // 4)" # TODO: handle values that depend on T
```

Comment:
Should we parse the string on instantiation of `FOMOParameterSet` ?

Reply:
Something like this. Actually, should parameters be function of `T` by default? Or even better should they be function of `nlp`?